eclipsePlugin/src/de/tobject/findbugs/io/FileOutput.java

```diff
@@ -21,14 +21,11 @@
 import java.io.IOException;
 import java.io.OutputStream;
 
-import org.eclipse.core.resources.IFile;
-import org.eclipse.core.runtime.IProgressMonitor;
```

Comment:
yes we having bugs, like everybody else, but theres an trick so workaround this. @hazendaz



@timtebeek again the import bug can not find it to link, assuming ticket is closed again.

Reply:
Did you mean this one? Not clear what you're tagging me for here.
- https://github.com/openrewrite/rewrite/pull/5738

Reply:
yes thanks this is what i meant. Its fixed in the code but still appears in prod. So this might be a thing to investigate if its not done by some version bump, which should be the case.

Reply:
That pull request is still open and out for review; not yet released or available. Not sure what you expected here?
Best not to open PRs without verifying the results yourself locally, especially when you know your fix hasn't made it in yet.